### PR TITLE
Alignment snippets

### PR DESCRIPTION
**Pull request by joshuak94**
Contains Copy'n'Pastable alignment snippets.

### CODE REVIEW

**Comment by marehr** [comment]
Thank you :+1:

File: test/snippet/alignment/configuration/align_cfg_aligned_ends.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//! [aligned_ends]
```
Comment (marehr) [inline test/snippet/alignment/configuration/align_cfg_aligned_ends.cpp]: can this be removed?

File: test/snippet/alignment/configuration/align_cfg_aligned_ends_access.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
+//! [access]
```
Comment (marehr) [inline test/snippet/alignment/configuration/align_cfg_aligned_ends_access.cpp]: can this be removed?

File: test/snippet/alignment/configuration/align_cfg_aligned_ends.cpp (hunk 1 above)
Comment (joshuak94) [reply test/snippet/alignment/configuration/align_cfg_aligned_ends.cpp]: Done!

File: test/snippet/alignment/configuration/align_cfg_aligned_ends_access.cpp (hunk 2 above)
Comment (joshuak94) [reply test/snippet/alignment/configuration/align_cfg_aligned_ends_access.cpp]: Also done!

**Pull request merged**